Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.yml

```diff
@@ -0,0 +1,504 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: XFE_v2
+  version: -1
+configuration:
+- defaultvalue: https://api.xforce.ibmcloud.com
+  display: Server URL (e.g. https://api.xforce.ibmcloud.com)
+  name: url
+  required: true
+  type: 0
+- display: API Key
+  name: credentials
+  required: true
+  type: 9
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: '7'
+  display: IP Threshold. Minimum risk score for the IP to be consodered malicious
+    (ranges from 1 to 10).
+  hidden: false
+  name: ip_threshold
+  required: false
+  type: 0
+- defaultvalue: '7'
+  display: URL Threshold. Minimum risk score for the URL to be consodered malicious
+    (ranges from 1 to 10).
+  hidden: false
+  name: url_threshold
+  required: false
+  type: 0
+- defaultvalue: '5'
+  display: CVE Threshold. Minimum risk score for the URL to be consodered malicious
+    (ranges from 1 to 10).
+  hidden: false
+  name: cve_threshold
+  required: false
+  type: 0
+description: IBM X-Force Exchange lets you receive threat intelligence about applications,
+  IP addresses, URls and hashes
+display: IBM X-Force Exchange (v2)
+name: XFE_v2
+script:
+  commands:
+  - arguments:
+    - default: true
+      description: IP to check
+      isArray: false
+      name: ip
+      required: true
+      secret: false
+    - default: false
+      description: 'score treshold '
+      isArray: false
+      name: threshold
+      required: false
+      secret: false
+    - default: false
+      defaultValue: 'false'
+      description: Should we return full response
+      isArray: false
+      name: long
+      required: false
+      secret: false
+    deprecated: false
+    description: IP to check
+    execution: false
+    name: ip
+    outputs:
+    - contextPath: IP.Address
+      description: Bad IP found
+      type: Unknown
+    - contextPath: IP.Malicious.Vendor
+      description: For malicious IPs, the vendor that made the decision
+      type: Unknown
+    - contextPath: IP.Malicious.Description
+      description: For malicious IPs, the reason for the vendor to make the decision
+      type: Unknown
+    - contextPath: IP.Malicious.Score
+      description: For malicious IPs, the score from the vendor
+      type: Unknown
+    - contextPath: DBotScore.Indicator
+      description: The indicator we tested
+      type: Unknown
+    - contextPath: DBotScore.Type
+      description: The type of the indicator
+      type: String
+    - contextPath: DBotScore.Vendor
+      description: Vendor used to calculate the score
+      type: Unknown
+    - contextPath: DBotScore.Score
+      description: The actual score
+      type: Unknown
+    - contextPath: XFE.IP(obj.Address==val.Address).Reason
```

Comment:
in the yml you shouldn't mention `(obj.Address==val.Address)` just in the code.
please change in the yml and readme
`XFE.IP(obj.Address==val.Address).Reason` => `XFE.IP.Reason`